### PR TITLE
WIP: support for ship-it v2

### PR DESCRIPTION
**Pull request by rail**

### CODE REVIEW

**Comment by tomprince** [comment]
Does this need to be a different action type, or can which version of the api to talk to be a scriptworker level configuration?

**Comment by rail** [comment]
I thought that it'd be easier to remove v1 related code (mark-as-shipped and mark-as-started) in the future this way, even though it looks suboptimal.

**Comment by rail** [comment]
The travis result is expected to be red until https://github.com/mozilla-releng/shipitapi/pull/10 is merged and the corresponding version is released.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+# [3.0.0] - 2018-11-19
+### Fixed
```
Comment (MihaiTabara) [inline CHANGELOG.md]: Nit: s,`Fixed`,`Added` ?

File: SCOPES.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ Supported scope prefixes:
   * **Conflicts**: with any other `{scope_prefix}:action:*`
   * **Branch Restrictions**: `None`
 
+* `{scope_prefix}:action:mark-as-shipped-v2`
```
Comment (MihaiTabara) [inline SCOPES.md]: 👍

File: shipitscript/ship_actions.py [hunk 3]
```diff
@@ -25,6 +25,22 @@ def mark_as_shipped(ship_it_instance_config, release_name):
                              status='shipped', shippedAt=shipped_at)
 
 
+def mark_as_shipped_v2(ship_it_instance_config, release_name):
+    """Function to make a simple call to Ship-it API v2 to change a release
+    status to 'shipped'
+    """
+    taskcluster_client_id, taskcluster_access_token, api_root, \
```
Comment (MihaiTabara) [inline shipitscript/ship_actions.py]: Nit: maybe we can we gather all this within a tuple that we can split across two lines to improve readability? Something like

```
(taskcluster_client_id, taskcluster_access_token, 
  api_root, timeout_in_seconds) = get_auth_primitives_v2(ship_it_instance_config)
```

File: shipitscript/ship_actions.py [hunk 4]
```diff
@@ -25,6 +25,22 @@ def mark_as_shipped(ship_it_instance_config, release_name):
...
+        timeout_in_seconds = get_auth_primitives_v2(ship_it_instance_config)
+    release_api = shipitapi.Release_V2(
+            taskcluster_client_id=taskcluster_client_id,
+            taskcluster_access_token=taskcluster_access_token,
+            api_root=api_root, timeout=timeout_in_seconds)
+
+    log.info('Marking the release as shipped...')
+    release_api.update_status(release_name, status='shipped')
+    check_release_has_values_v2(release_api, release_name, status='shipped')
```
Comment (MihaiTabara) [inline shipitscript/ship_actions.py]: ++ for adding the post-update check!

**Pull request merged**